vowpalwabbit/interactions.cc

```diff
@@ -154,21 +154,6 @@ void eval_count_of_generated_ft(bool permutations, const std::vector<std::vector
   }
 }
 
-bool sort_interactions_comparator(const std::vector<namespace_index>& a, const std::vector<namespace_index>& b)
-{
-  if (a.size() != b.size()) { return a.size() > b.size(); }
```

Comment:
Is this desired behavior? If so, then we'll need a comparator that does this bit